src/button/button-group.scss

```diff
@@ -7,77 +7,55 @@
   align-items: center;
   isolation: isolate;
 
-  $childSelector: 'input, button'; // target nested inputs and buttons
+  $childZindexSelector: '.iui-input-container, .iui-button, .iui-input:where(:not(.iui-input-container .iui-input))';
+  $childBorderSelector: '.iui-input, .iui-button:not(.iui-borderless)';
+  $disabledSelector: '&:disabled, &.iui-disabled, &[aria-disabled="true"]';
 
   > * {
     display: flex;
 
-    #{$childSelector} {
+    #{$childZindexSelector} {
       position: relative;
 
       &:hover {
         z-index: 1;
       }
 
-      &:focus {
+      &:focus,
+      &:focus-within {
         z-index: 2;
-
-        &:where(:not(:focus-visible):not(:hover)) {
-          z-index: revert;
-        }
       }
 
-      &:disabled {
+      #{$disabledSelector} {
         z-index: -1;
       }
 
+      // Adds stripe above active button
+      --_active-stripe-inset: #{$iui-xxs} #{$iui-xxs} calc(100% - #{$iui-xs});
```

Comment:
So once again we are removing functionality from IE...

Reply:
The code and specificity benefit seems worth it to me